docs/man_pages/project/testing/debug.md

```diff
@@ -9,6 +9,24 @@ position: 6
 
 Initiates a debugging session for your project on a connected device or native emulator. <% if(isMacOS) { %>You must specify the target platform on which you want to debug.<% } %> When necessary, the command will prepare, build, deploy and launch the app before starting the debug session. While debugging, the output from the application is printed in the console and any changes made to your code are synchronized on all connected devices or running emulators.
 
+<% if(isHtml) { %>
+#### How file changes are handled
+With HMR (Hot Module Replacement):
+* Changes in `.js`, `.ts`, `.less`, `.sass` and other file types that are accepted will cause a refresh of the application.
+* Changes in `App_Resources` will cause a rebuild of the application.
+* Changes in `node_modules/somePlugin` if accepted will cause a refresh of the application.
+* Changes in `node_modules/somePlugin/platforms` will cause a rebuild of the application.
```

Comment:
We can add here that the `node_modules/somePlugin/package.json` file is watched as well and considered as native change.